odd-collector/odd_collector/adapters/postgresql/mappers/relationships.py

```diff
@@ -0,0 +1,54 @@
+from odd_collector.adapters.postgresql.models import Relationship
+from odd_models import DataEntity, DataEntityType, ERDRelationship
+from odd_models import Relationship as DataEntityRelationship
+from odd_models import RelationshipType
+from oddrn_generator import PostgresqlGenerator
+
+
+def map_relationship(
+    generator: PostgresqlGenerator,
+    relationship: Relationship,
+    table_entities: dict[str, DataEntity],
+) -> DataEntity:
+    generator.set_oddrn_paths(
+        **{
+            "schemas": relationship.schema_name,
+            "tables": relationship.table_name,
+            "relationships": (
+                f"references_{relationship.referenced_table_name}_with_{relationship.constraint_name}"
+            ),
+        }
+    )
```

Comment:
I don't mind but is there any particular reason here to use dict unpacking? Can we use default arg names? It looks a bit weird and unnecessary.
```suggestion
generator.set_oddrn_paths(
  schemas=relationship.schema_name,
  tables=relationship.table_name,
  relationships=(
    f"references_{relationship.referenced_table_name}_with_{relationship.constraint_name}"
  )
)
```

Reply:
Agree, I didn't pay attention to this because in other adapter that redundant logic was used.